src/Explorer/Tabs/TabsBase.ts

```diff
@@ -61,6 +63,10 @@ export default class TabsBase extends WaitsForTemplateViewModel {
     };
   }
 
+  public getIndex(): number {
```

Comment:
No blocking nit. You don't need this method. Mark the `index` property `public readonly` and you can only write it in the constructor.

Reply:
nice catch, will change it.